lib/improver/utilities/cube_checker.py

```diff
@@ -171,3 +171,49 @@ def find_dimension_coordinate_mismatch(
     else:
         mismatch = list(set(second_dim_names) - set(first_dim_names))
     return mismatch
+
+
+def coord_exists_in_cube(cube, coord_name):
+    """
```

Comment:
Where are the unit tests for this function

Reply:
This has been removed as no longer used